packages/web-components/src/global/main.css

```diff
@@ -8,4 +8,4 @@ va-process-list li :is(h2, h3, h4, h5) {
   margin-top: 0;
   clear: none;
   padding-top: 0.3rem;
-}
+}
```

Comment:
Is it possible to add the line break back to this file so that it's not showing a change here? If not, no big deal.